libraries/loader/package-lock.json

```diff
@@ -0,0 +1,96 @@
+{
```

Comment:
This file was created by using `npm` instead of `yarn` here.  You should be able to just delete the file.  You may also need to run `yarn` in the root directory of the repository in order to fix the `yarn.lock` file contents.